src/main/java/org/reaktivity/nukleus/http/internal/routable/stream/TargetOutputEstablishedStreamFactory.java

```diff
@@ -239,14 +235,14 @@ private void processBegin(
                     headers = headers0;
                 }
 
-                this.sourceId = newSourceId;
-                this.target = newTarget;
-                this.targetId = newTargetId;
+                if (!targetStream.started)
+                {
+                    target.doBegin(targetStream.streamId, 0L, correlation.id());
+                    targetStream.started = true;
+                }
+                target.setThrottle(targetStream.streamId, this::handleThrottle);
```

Comment:
This throttle handler seems to be dealing with `WINDOW` updates after writing encoded response.
What happens if a `WINDOW` frame arrives after the encoded response is completed?